packages/editor/src/components/block-icon/index.js

```diff
@@ -6,19 +6,24 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { Dashicon, SVG } from '@wordpress/components';
+import { Dashicon, Path, SVG } from '@wordpress/components';
 import { createElement, Component } from '@wordpress/element';
 
 function renderIcon( icon ) {
 	if ( 'string' === typeof icon ) {
+		if ( icon === 'block-default' ) {
```

Comment:
@jasmussen - is it okay to handle SVG replacement in here?

Reply:
Can you elaborate a little bit? Not sure I understand the context.

Reply:
See also my previous comment: https://github.com/WordPress/gutenberg/pull/10652#issuecomment-430682576.

Reply:
I changed my mind when entertaining with the exisitng implementation. I reverted all change in `normalizeIconObject` to use `default-block` dashicon. However, it is replaced with the new nice `SVG` icon inside `BlockEdit`. This should behave exactly the same in the UI.

Reply:
Ah, gotcha. I don't have strong opinions here, and I would defer to you two for the technical implementation. What matters is:

- We encourage the use of SVG (i.e. make it easy)
- We can provide an icon set even for new blocks, like Dashicons, in the future.
- 3rd party provided SVGs are processed so `role` and `focusable` attributes are added.

If those values are upheld, 👍 👍 from me.

Reply:
> We can provide an icon set even for new blocks, like Dashicons, in the future.

I think it would be convenient to be able to use strings if someone is happy with the default icon set.

> 3rd party provided SVGs are processed so role and focusable attributes are added.

This is what `SVG` component does :)